### PR TITLE
Fix/issue-28: verbose logging checkbox for creating new experiment  and existing experiment

### PR DESCRIPTION
**Pull request by ppratikcr7**
This will give verbose logging on /assign call on server side.

Screenshot 2021-05-03 at 10 58 24 AM

### CODE REVIEW

**Comment by ppratikcr7** [comment]
We need to push another commit in this to optimize a code logic. We will merge it after that.

**Review by msandbothe — APPROVED** [review]

File: frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts [hunk 1]
```diff
@@ -145,7 +148,17 @@ export class ViewExperimentComponent implements OnInit, OnDestroy {
   get DialogType() {
     return DialogType;
   }
-
+  
+  toggleLogging(e) {
```
[the remark this answers is not in the document]
Comment (ppratikcr7) [reply frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts:152]: Okay. Will make the changes.

File: frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts [hunk 2]
```diff
@@ -145,7 +148,17 @@ export class ViewExperimentComponent implements OnInit, OnDestroy {
...
+    if (e.checked) {
```
[the remark this answers is not in the document]
Comment (ppratikcr7) [reply frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts:153]: Okay sure

File: frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts [hunk 3]
```diff
@@ -48,6 +48,8 @@ export class ViewExperimentComponent implements OnInit, OnDestroy {
     private router: Router,
     private _snackBar: MatSnackBar
   ) {}
+  
+  verboseLogging: boolean;
```
[the remark this answers is not in the document]
Comment (ppratikcr7) [reply frontend/projects/abtesting/src/app/features/dashboard/home/pages/view-experiment/view-experiment.component.ts:52]: okay done

**Comment by ppratikcr7** [comment]
Changes requested are done. We can merge the PR now.

**Pull request merged**